test/backend/test_node.py

```diff
@@ -1899,6 +1902,102 @@ def test_average_pool_3d(self):
                        strides=strides,
                        pooling_type="AVG")
 
+  def test_lp2_pool_2d(self):
+    kernel_shape = [1, 2]
+    strides = [1, 2]
+    p = 2
+
+    input_shape = [10, 10, 4, 4]
+    self._test_pooling(input_shape=input_shape,
+                       kernel_shape=kernel_shape,
+                       strides=strides,
+                       p=p)
```

Comment:
Looks like pooling_type is missing so it will used the default "MaxPool". Same for other lp test cases.

Reply:
Thanks for catching this. Fixed in the new commit.